src/ontodia/viewUtils/toSvg.ts

```diff
@@ -143,25 +199,31 @@ function clonePaperSvg(options: ToSVGOptions, elementSizePadding: number): {
         if (!overlayedView) { continue; }
 
         const elementRoot = document.createElementNS(SVG_NAMESPACE, 'g');
+        const overlayedViewContent = overlayedView.firstChild.cloneNode(true) as HTMLElement;
         elementRoot.setAttribute('class', 'ontodia-exported-element');
 
-        const newRoot = document.createElementNS(SVG_NAMESPACE, 'foreignObject');
+        let newRoot;
+        if (!isIE11()) {
```

Comment:
Please, switch `then` and `else` blocks to remove negation from condition expression.